go.mod

```diff
@@ -107,13 +107,13 @@ require (
 	go.uber.org/atomic v1.7.0 // indirect
 	go.uber.org/multierr v1.6.0 // indirect
 	go.uber.org/zap v1.19.0 // indirect
-	golang.org/x/crypto v0.16.0 // indirect
+	golang.org/x/crypto v0.22.0 // indirect
 	golang.org/x/mod v0.14.0 // indirect
-	golang.org/x/net v0.19.0 // indirect
+	golang.org/x/net v0.24.0 // indirect
```

Comment:
kube in 1.28 uses /x/net in v0.23.0 https://github.com/kubernetes/kubernetes/commit/adf10d3c3941a3d0decd9bc1a6a26fa6a864ec5e

v0.24.0 should be backward compatible.